CHANGELOG.rst

```diff
@@ -3,6 +3,13 @@
 salt-formula
 ============
 
+0.0.4
```

Comment:
And don't forget that this new 0.0.4 version also contains all commits since 0.0.3 - so many fixes and stuff you should also mention here.